payment-element/server/dotnet/server.csproj

```diff
@@ -1,20 +1,11 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
-
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp3.1;net5.0</TargetFrameworks>
+    <TargetFrameworks>net6.0</TargetFrameworks>

```

Comment:
Wonder if we should update the README to show:

```bash
dotnet run Program.cs --framework=net6.0
```

Maybe we don't need the explicit version after this?

Reply:
we can should probably just target one framework. 6.0 is an LTS release